### PR TITLE
Consolidate and tidy tools page

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@chaudum Made a few more changes than expected, but nice and tidy now… I think.

### CODE REVIEW

File: pages/docs/clients/index.html [hunk 1]
```diff
@@ -45,6 +38,10 @@
...
 
-## Utilities and Data Visualization Tools
+### Utilities and Data Visualization Tools
+
+For a full list of tools, see [Tools Overview]({% url '/docs/tools/index.html' %}).
+
+## Next Steps
 
-For a full list of tools see [Tools Overview]({% url '/docs/tools/index.html' %}).
+- [Scale a Cluster](/docs/scale)
```
Comment (chaudum) [inline pages/docs/clients/index.html]: please use `{% url '...' %}` for cactus internal links.

File: pages/docs/tools/index.html [hunk 2]
```diff
@@ -2,42 +2,45 @@
...
-There is a setup guide about [how to connect to Crate with SQirreL]({% url '/docs/tools/squirrel.html' %}).
+Here is a setup guide about [how to connect to Crate with SQirreL]({% url '/docs/tools/squirrel.html' %}).
 
-[Download](http://squirrel-sql.sourceforge.net), [Official documentation](http://squirrel-sql.sourceforge.net/paper/SQuirreL_us.pdf)
+- [Download](http://squirrel-sql.sourceforge.net)
+- [Official documentation](http://squirrel-sql.sourceforge.net/paper/SQuirreL_us.pdf)
 
 ### conZole
-conZole is a web-based SQL console written in NodeJS that also lets you inspect the date in your Crate. It uses the [node-crate](https://github.com/megastef/node-crate) driver and [Zebra UI](http://www.zebkit.com/) as the HTML5 frontend.
+conZole is a web-based SQL console written in Node.JS that lets you inspect data in a Crate cluster. It uses the [node-crate](https://github.com/megastef/node-crate) driver and [Zebra UI](http://www.zebkit.com/) as the HTML5 front end.
```
Comment (chaudum) [inline pages/docs/tools/index.html]: seems to be spelled: Node.js (had to look it up myself)

File: pages/docs/tools/index.html [hunk 3]
```diff
@@ -2,42 +2,45 @@
...
-
-[Github](https://github.com/crate/crash), [Download from PyPi](https://pypi.python.org/pypi/crash)
+- How to connect to Crate with crash [here](https://crate.io/docs/connect/crash/)
+- Full reference [documentation](https://crate.io/docs/reference/crash/)
+- [Github](https://github.com/crate/crash)
+- [PyPi](https://pypi.python.org/pypi/crash)
 
 ### cr8 Utilities
-`cr8` is a collection of command line tools for Crate written in Python. If makes repeating tasks such as filling tables with sample data really easy, thus being a Swiss army knife for Crate developers. It is develped by one of our core developers and has been featured on our blog: [A Collection of Utility Scripts for Working With Clusters](https://crate.io/a/cr8-utility-scripts/)
+`cr8` is a collection of command line tools for Crate written in Python and developed by one of our core developers. It reduces repetitive tasks such as filling tables with sample data, a real 'Swiss army knife' for Crate users. It's developed by one of our core developers
```
Comment (chaudum) [inline pages/docs/tools/index.html]: the 2nd sentence is repeated

File: pages/docs/tools/index.html [hunk 4]
```diff
@@ -46,20 +49,22 @@
...
-Although Kibana is product for Elasticsearch, it seems to work also nicely with Crate. Stefan Thies wrote about [How to use Kibana with Crate](http://techblog.bigdata-analyst.de/2015/06/25/how-to-use-kibana-4-with-crate-io/)
+Kibana is an open-source data visualization platform that allows you to visualize and interact with your data. Although Kibana is product for Elasticsearch, it works well with Crate. Stefan Thies wrote about [How to use Kibana with Crate](http://techblog.bigdata-analyst.de/2015/06/25/how-to-use-kibana-4-with-crate-io/)
 
-[Website](https://www.elastic.co/products/kibana)
+- [Website](https://www.elastic.co/products/kibana)
 
 ### SQLPad
-SQLPad is a beautiful, web-based SQL editor written in NodeJs that can be run in your own private cloud and supports multiple databases such as Postgres, MySQL, SQL Server, Vertica and recently (since version `1.12.0`) also Crate.
+SQLPad is a beautiful, web-based SQL editor written in Node.Js that can run in your own private cloud and supports multiple databases such as Postgres, MySQL, SQL Server, Vertica and since version `1.12.0`, also Crate.
```
Comment (chaudum) [inline pages/docs/tools/index.html]: Node.js

**Comment by ChrisChinchilla** [comment]
OK, fixed up @chaudum

**Comment by ChrisChinchilla** [comment]
OK, think that's done for this PR @chaudum likely needed in other places too, but can cover when needed.

**Comment by chaudum** [comment]
ack

**Pull request merged**